### ISSUE TITLE
hover-to-animate emoji not clear (accessibility

### ISSUE THREAD

**Issue by hoodieak** [unascribed/sleeping.town#45]
a little ⏯ sort of emoji or visual cue beside animate hover emoji would be Wonderful for accessibility, thank yu,

**Comment by unascribed** [comment]
should probably extend this to all things that can be played on hover

**Comment by hoodieak** [comment]
yeah but the emoji are the biggest one for me at least? we already know about gif avis from having been here long enough at least, yknow?
in general at least

**Comment by unascribed** [comment]
well, that's only true for people who've Been Around on mastodon

lots of people don't know animated avatars are valid

**Comment by hoodieak** [comment]
fair
